src/pages/ProductDetailsPage/ProductDetailsPage.module.scss

```diff
@@ -1,7 +1,280 @@
-.footer {
```

Comment:
Divided modifiers into different modules, plz. By the rules, each module has its own modifier 

ex: ItemCard.module.scss, Gallery.module.scss